### ISSUE TITLE
Fake deviceMemory not detected on Firefox

### ISSUE THREAD

**Issue by JobcenterTycoon** [abrahamjuliot/creepjs#269]
When i set the value with uBlock Origin with `abrahamjuliot.github.io##+js(set, navigator.deviceMemory, 1)` on Firefox the site doesn’t detect this fake and also clears my visits.

Firefox doesn’t support `navigator.deviceMemory` https://developer.mozilla.org/en-US/docs/Web/API/Navigator/deviceMemory

**Comment by abrahamjuliot** [comment]
Good catch. It's detected in `console.log(Fingerprint.navigator.lied)` but not shown in the top section. There's a good handful of similar existing inline detections that can be shown with a message. 

You should get two different fingerprints, one clean and one wrong. The invalid one should not change if you rotate the memory via JS.

**Comment by JobcenterTycoon** [comment]
Yes 2 fingerprints and no change when i rotate. But a fake memory on a browser which don’t even support this should be just throwed away without affecting the final fingerprint or not?

**Comment by abrahamjuliot** [comment]
It's better to preserve the additional entropy if we can keep it stable. The existing lie is better entropy than a restored result. Maybe there are some use cases for restoring the original, but they will be limited to what we can restore.
